From b228347d62b7c69f16d1983d4c8c70f6601c7351 Mon Sep 17 00:00:00 2001
From: hololeap <hololeap@users.noreply.github.com>
Date: Sun, 2 Jul 2023 14:33:27 -0600
Subject: [PATCH] Change dependency on Cabal-syntax

Do not add Cabal-syntax to build-depends when the Cabal-syntax flag is
off.

This will ensure that Cabal-syntax is ignored/omitted when the flag is
off which will prevent version conflicts with >=Cabal-syntax-3.8

Signed-off-by: hololeap <hololeap@users.noreply.github.com>
---
 hackage-security.cabal | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/hackage-security.cabal b/hackage-security.cabal
index 58afd57..f0463f4 100644
--- a/hackage-security.cabal
+++ b/hackage-security.cabal
@@ -170,7 +170,6 @@ library
   else
     build-depends: Cabal        >= 2.2.0.1 && < 2.6
-                             || >= 3.0     && < 3.7,
-                   Cabal-syntax <  3.7
+                             || >= 3.0     && < 3.7
 
   hs-source-dirs:      src
   default-language:    Haskell2010
